src/components/Nuttakit.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <div class="container">
+    <div class="columns">
+      <div class="columns is-narrow">
+        <img src="https://i.pinimg.com/originals/95/13/5e/95135e0281c333a034c004488c05c7e1.jpg" alt="" width="200px">
+      </div>
+      <div class="collumns is-narrow">
+        <h1 class="title is-1" style=""> &nbsp&nbsp Guide for bodybuilders </h1>
+      </div>
+    </div>
+    <input type="text" v-if="results.length >= 0" v-model="q" class="input-search" placeholder="Search Engine">
+    <div v-if="isPlayersNotFound" class="has-text-centered">
+      <h1 class="title is-1">☹️</h1>
+      <h1 class="title is-1">Not Found!</h1>
+      <br><br><br><br><br><br><br><br>
+    </div>
+    <div class="columns is-multiline">
+      <div v-for="result in filterMuscles" :key="result.id" class="column is-3">
+
+          <div class="player">
+
+            <div class="player-name">
+              <a :href="`//en.wikipedia.org/wiki/${result.name}`" target="_blank">
```

Comment:
ข้อมูลซ้ำกันพวกนี้ทำไมไม่เอาไปเก็บใน json แล้วใช้ v-for วนดึงค่าออกมาแสดงเอาครับ